File: ncDados/dMunicipios.vb
Namespace nsMunicipios

  Public Class ColecaoMunicipios
    Inherits List(Of dMunicipios)
  End Class

  Public Class dMunicipios

    Private _cid As Nullable(Of Integer)
    Private _codigo_ibge As String
    Private _nome As String
    Private _estados_cid As Integer
    Private _situacao As String

    Public Property cid() As Nullable(Of Integer)
      Get
        Return _cid
      End Get
      Set(ByVal value As Nullable(Of Integer))
        _cid = value
      End Set
    End Property

    Public Property codigo_ibge() As String
      Get
        Return _codigo_ibge
      End Get
      Set(ByVal value As String)
        _codigo_ibge = value
      End Set
    End Property

    Public Property nome() As String
      Get
        Return _nome
      End Get
      Set(ByVal value As String)
        _nome = value
      End Set
    End Property

    Public Property estados_cid() As Integer
      Get
        Return _estados_cid
      End Get
      Set(ByVal value As Integer)
        _estados_cid = value
      End Set
    End Property

    Public Property situacao() As String
      Get
        Return _situacao
      End Get
      Set(ByVal value As String)
        _situacao = value
      End Set
    End Property

  End Class

End Namespace

File: ncModelos/dMunicipios.cs
using System;
using System.Collections.Generic;

namespace ncModelos
{
    public class ColecaoMunicipios : List<dMunicipios>
    {
    }

    public class dMunicipios
    {
        private int? _cid;
        private string _codigo_ibge;
        private string _nome;
        private int _estados_cid;
        private string _situacao;

        public int? cid
        {
            get => _cid;
            set => _cid = value;
        }

        public string codigo_ibge
        {
            get => _codigo_ibge;
            set => _codigo_ibge = value;
        }

        public string nome
        {
            get => _nome;
            set => _nome = value;
        }

        public int estados_cid
        {
            get => _estados_cid;
            set => _estados_cid = value;
        }

        public string situacao
        {
            get => _situacao;
            set => _situacao = value;
        }
    }

}
